ui/src/components/UI/SyntaxHighlighter/index.tsx

```diff
@@ -1,30 +1,51 @@
 import React from 'react';
-import {Prism as SyntaxHighlighterContainer} from 'react-syntax-highlighter';
-import {highlighterStyle} from './highlighterStyle'
+import Lowlight from 'react-lowlight'
+import 'highlight.js/styles/atom-one-light.css'
 import './index.scss';
 
+import xml from 'highlight.js/lib/languages/xml'
+import json from 'highlight.js/lib/languages/json'
+import protobuf from 'highlight.js/lib/languages/protobuf'
+import javascript from 'highlight.js/lib/languages/javascript'
+import actionscript from 'highlight.js/lib/languages/actionscript'
+import wasm from 'highlight.js/lib/languages/wasm'
+import handlebars from 'highlight.js/lib/languages/handlebars'
+import yaml from 'highlight.js/lib/languages/yaml'
+import python from 'highlight.js/lib/languages/python'
+
+Lowlight.registerLanguage('python', python);
+Lowlight.registerLanguage('xml', xml);
+Lowlight.registerLanguage('json', json);
+Lowlight.registerLanguage('yaml', yaml);
+Lowlight.registerLanguage('protobuf', protobuf);
+Lowlight.registerLanguage('javascript', javascript);
+Lowlight.registerLanguage('actionscript', actionscript);
+Lowlight.registerLanguage('wasm', wasm);
+Lowlight.registerLanguage('handlebars', handlebars);
+
 interface Props {
     code: string;
-    style?: any;
     showLineNumbers?: boolean;
-    className?: string;
     language?: string;
-    isWrapped?: boolean;
 }
 
 export const SyntaxHighlighter: React.FC<Props> = ({
-                                                code,
-                                                style = highlighterStyle,
-                                                showLineNumbers = true,
-                                                className,
-                                                language = 'python',
-                                                isWrapped = false,
-                                            }) => {
-    return <div className={`highlighterContainer ${className ? className : ''} ${isWrapped ? 'wrapped' : ''}`}>
-        <SyntaxHighlighterContainer language={language} style={style} showLineNumbers={showLineNumbers}>
-            {code ?? ""}
-        </SyntaxHighlighterContainer>
-    </div>;
+        code,
+        showLineNumbers = false,
+        language = null
+    }) => {
+    let arr = code.split("\n");
+    const markers = [];
+
+    if (showLineNumbers) {
+        for (var i = 0; i < arr.length; i++) {
+            markers.push({
+                line: i + 1,
+                className: 'hljs-marker-line'
+            });
+        }
+    }
```

Comment:
I would replace lines 37-47 with this, wdyt?
```
const arr = code.split("\n");
const markers = showLineNumbers ? arr.map((item, i) => {return {
                line: i + 1,
                className: 'hljs-marker-line'
            }}) : [];
```

Reply:
@lirazyehezkel done with https://github.com/up9inc/mizu/commit/e2299c831c009565e476c5be3fdeb62006535091